mm2src/mm2_bin_lib/src/mm2_native_lib.rs

```diff
@@ -177,8 +204,8 @@ pub extern "C" fn mm2_test(torch: i32, log_cb: extern "C" fn(line: *const c_char
         log!("mm2_test] Restarting MM…");
         let conf = CString::new(&conf[..]).unwrap();
         let rc = unsafe { mm2_main(conf.as_ptr(), log_cb) };
-        let rc = MainErr::from_i8(rc).unwrap();
-        if rc != MainErr::Ok {
+        let rc = StartupErrorCode::from_i8(rc).unwrap();
+        if rc != StartupErrorCode::Ok {
             log!("!mm2_main: {:?}", rc);
             return -1;
```

Comment:
Should we handle this more strictly from `StartupErrorCode` instead of hard-coding `-1` everywhere?

Reply:
Fixed, kept `-1` in the places where the error is not a startup one.